src/platform/zc702/pl.dtsi

```diff
@@ -0,0 +1,9 @@
+/*
+ * CAUTION: This file is automatically generated by Xilinx.
+ * Version:
+ * Today is: Tue Aug 20 07:12:28 2019
+ */
+
+
+/ {
+};
```

Comment:
Hi Min,
the default pl.dtsi generated by petalinux will have the axi_intc node. The same node is present in the system-user.dtsi file. I want to overwrite the default pl.dtsi file with this empty file so that petalinux uses axi intc dts node from system-user.dtsi file